indico/modules/events/registration/templates/display/conference_home.html

```diff
@@ -43,7 +43,8 @@
         {% else %}
             <div>{% trans %}Registration for this event is currently open.{% endtrans %}</div>
             <div class="toolbar right">
-                <a href="{{ url_for('event_registration.display_regform_list', event) }}"
+                {#- _external=True needed to avoid broken link on the offline website -#}
+                <a href="{{ url_for('event_registration.display_regform_list', event, _external=True) }}"
```

Comment:
`true`

And I wonder if it wouldn't make more sense to hide the registration stuff in the static site. cc @pferreir